lib/python/rose/macro.py

```diff
@@ -435,7 +435,7 @@ def get_id_from_section_option(section, option):
 
 
 def load_meta_path(config=None, directory=None, is_upgrade=False,
-                   locator=None):
+                   locator=None, opt_meta_paths=[]):
```

Comment:
Declaring lists in a function definition can lead to weird behaviour - even though this is used in a safe way,
can we have something like this?
```
locator=None, opt_meta_paths=None):
...
...
if opt_meta_paths is None:
    opt_meta_paths = []
```
and the same in load_meta_config_tree.